docs/migration/migrate_7_0.asciidoc

```diff
@@ -49,4 +49,11 @@ to Kibana's advanced setting.
 
 *Impact:* This setting is no longer necessary. If you enable {monitoring} across the Elastic Stack, a monitoring agent runs
 on each Elasticsearch node, Logstash node, Kibana instance, and Beat to collect and index metrics. Each node and instance is
-considered unique based on its persistent UUID, which is written to the path.data directory when the node or instance starts.
\ No newline at end of file
+considered unique based on its persistent UUID, which is written to the path.data directory when the node or instance starts.
+
+[float]
+=== kibana.yml setting `elasticsearch.url` sets port based on protocol
+*Details:* In prior versions of Kibana, if no port was specified in `elasticsearch.url` a default of 9200 was chosen.
+The port is now protocol dependent: https ports will use 443, and http ports will use 80.
```

Comment:
> The port is now protocol dependent

It's not necessarily dependent on the protocol, as we default to 9200 if a protocol is not provided. How about "The port is now based off the protocol, if provided."